transformer.md

```diff
@@ -255,3 +255,62 @@ The end marker is the following, where _`<N>`_ is the number of the step:
 ```markdown
 <!-- Killercoda step<N>.md END -->
 ```
+
+## Generate a tutorial
+
+You can generate a new tutorial from existing documentation using the transformer tool.
+After generation, a [GitHub Actions workflow](./.github/workflows/regenerate-tutorials) updates the tutorial when the documentation source changes.
+
+### Before you begin
+
+1. Clone the repository with the source documentation.
+1. Clone the Killercoda repository.
+1. [Download and install Go](https://go.dev/doc/install) to run the transformer tool.
+
+To generate a tutorial:
+
+1. In the source repository, check out a new branch.
+
+   Give the branch a name that reflects the planned changes.
+   For example, `2024-06-killercoda-tutorial-for-loki-quickstart`.
+
+1. In the Killercoda repository, check out a new branch.
+
+   For convenience, name the branch the same as you did in the source repository.
+
+1. In the source repository, run `make docs` from the `docs/` directory so that you can verify your changes don't break the rendered documentation.
+
+1. In the source file, add the Killercoda front matter.
+
+   For more information and an example front matter, refer to [Metadata](#Metadata)
+
+1. Add directives for each step in the tutorial.
+
+   Each step starts at the [step](#step) directive start marker and ends at the [step](#step) directive end marker.
+   Include at least one step.
+   The first step use the start marker `<!-- Killercoda step1.md START -->` and the end marker `<!-- Killercoda step1.md END -->`
+
+1. Generate the tutorial.
+
+   1. In the Killercoda repository, change to the `tools/transformer` directory.
+   1. Run `go run ./ <PATH TO SOURCE FILE> <PATH TO OUTPUT DIRECTORY>`
+
+      - _`<PATH TO SOURCE FILE>`_ is the path to the documentation file.
+
+        For example, `~/ext/grafana/loki/docs/sources/get-started/quick-start.md`.
+
+      - _`<PATH TO OUTPUT DIRECTORY>`_ is the path to the output directory in the Killercoda repository.
+
+        For example, `~/ext/grafana/killercoda/loki/loki-quickstart`.
```

Comment:
Should we also tell them before this step to create the directory `loki-quickstart` or can the tool now generate the directory on it own?

Reply:
With https://github.com/grafana/killercoda/pull/15, the tool is able to create them automatically and errors if it can't create the output directory.